<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ArquivoDeRecursos.qrc">:/imgs/imagens/puc-goias-logo-06AD67F0EA7-seeklogo.com.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ArquivoDeRecursos.qrc">:/imgs/imagens/9f6dffae-6464-4a52-bdd2-d10983f44ca0.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>-20</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(228, 235, 240);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Projeto Teste LLDE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInserirInicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Inserir no Início</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarInicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Acessar o Início</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>1031</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarInicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Retirar do Início</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInserirFim">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Inserir no Fim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarFim">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Acessar o Fim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarFim">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Retirar do Fim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInserirPos">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Inserir na Posição</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarPos">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Acessar a Posição</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarPos">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Retirar da Posição</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarPos_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ordenar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaOrdenada">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>1031</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista Ordenada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>260</y>
      <width>73</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(228, 235, 240);</string>
    </property>
    <property name="text">
     <string>Ordem</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>300</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Cresecente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrescente</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>141</y>
      <width>531</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(228, 235, 240);</string>
         </property>
         <property name="text">
          <string>Valor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditValor">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(228, 235, 240);</string>
       </property>
       <property name="text">
        <string>Posição</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPos">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>408</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: rgb(228, 235, 240);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Quantidade de Elementos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditQE">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ArquivoDeRecursos.qrc"/>
 </resources>
 <connections/>
</ui>
